sdk/core/azure_core_tracing_opentelemetry/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+//! Azure Core OpenTelemetry tracing integration.
+//!
+//! This crate provides OpenTelemetry distributed tracing support for the Azure SDK for Rust.
+//! It bridges the standardized typespec_client_core tracing traits with OpenTelemetry implementation,
+//! enabling automatic span creation, context propagation, and telemetry collection for Azure services.
+
+mod attributes;
+mod conventions;
+mod span;
+pub mod telemetry;
```

Comment:
Only putting this out there and honestly have no strong opinion, but should we standardize on "tracing" ubiquitously in public APIs ourselves? I appreciate this crate already has "telemetry" baked into the name, though, so I could swing either way.

Reply:
The `telemetry` module shouldn't be public - the only type exported from the OTel wrapper should be the `OpenTelemetryTracerProvider` which is exported from the root.